crates/bevy_hierarchy/src/hierarchy.rs

```diff
@@ -8,7 +8,7 @@ use bevy_ecs::{
     system::EntityCommands,
     world::{Command, DeferredWorld, EntityWorldMut, World},
 };
-use bevy_utils::tracing::debug;
+use log::debug;
```

Comment:
Why is log necessary if we have tracing?

Reply:
`tracing` doesn't work on certain platforms if they don't support atomic CAS, whereas `log` works on every platform. Most notable example is Raspberry Pi Pico and other embedded ARM CPUs.